### PR TITLE
Fixed Spring Boot tutorial modules

### PR DESCRIPTION
**Pull request by Preeticp**
Combined: 5.2. Reducing the Hello World project memory usage and 8.5. Reducing Spring Boot project memory usage to form a reusable module with conditionals>Reducing project memory usage.
Fixed minor style leftovers.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-147-3 is available for testing at http://fabric8-online-docs-app-pr-147-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-147-4 is available for testing at http://fabric8-online-docs-app-pr-147-fabric8-online-docs.badger.fabric8.io

File: docs/topics/modules/using_workspace_spring_boot_project.adoc [hunk 1]
```diff
@@ -40,14 +40,13 @@ image::bs_code_add.png[Code Added to Spring Boot Project]
 +
 . Save your changes using kbd:[Ctrl+s] (or kbd:[Cmd+s] for macOS).
 
-. Click the *run* option (image:tri_run.png[title="Run button"]). This sets up the Spring Boot server. Unlike the Vert.x Quickstart project, the hot deploy option is not available for Spring Boot projects. As a result, changes are not automatically deployed.
+. Click the *run* option from the Run drop-down(image:tri_run.png[title="Run button"]). This sets up the Spring Boot server. Unlike the Vert.x Quickstart project, the hot deploy option is not available for Spring Boot projects. As a result, changes are not automatically deployed.
```
Comment (rkratky) [inline docs/topics/modules/using_workspace_spring_boot_project.adoc]: "Unlike *in* the Vert.x project..."?

File: docs/topics/modules/stopping_existing_workspaces.adoc [hunk 2]
```diff
@@ -1,29 +1,30 @@
 [id="stopping_existing_workspaces"]
 = Stopping existing workspaces
 
-Before creating a workspace for your project, stop the workspace for your previous project. Each running workspace uses your OpenShift Online resources. The OpenShift Online Free Tier supports one running workspace at a time. As a result, you must stop running workspaces for older projects each time before creating a new workspace.
+Each running workspace uses your OpenShift Online resources. The OpenShift Online Free Tier supports one running workspace at a time.
+When you start your new workspace for the Spring Boot quick start, your first workspace is automatically stopped. If it is not stopped, you must stop the running workspace for the older project before proceeding further.
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply docs/topics/modules/stopping_existing_workspaces.adoc]: To demarcate the introduction paragraph from the procedure. Would it be better to add : [discrete]
== Procedure here?

File: docs/topics/modules/reducing_project_memory_usage.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
++
+image::deploymentyaml.png[Deployment.Yaml]
++
+endif::[]
+//end conditional
+. Copy the contents of the following file to your new YAML file: https://raw.githubusercontent.com/burrsutter/vertx-eventbus/master/src/main/fabric8/deployment.yml[deployment.yaml].
++
+IMPORTANT: Spaces are meaningful in YAML files. Ensure that the correct spacing is copied into your YAML file from the link.
++
+
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply docs/topics/modules/reducing_project_memory_usage.adoc]: Adding a line after a step which has images associated with it or with conditionals, made it feel easier to demarcate and felt less cluttered, but this is just my opinion. I can do away with it if it affects the rendering or the file size. Let me know.

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-147-5 is available for testing at http://fabric8-online-docs-app-pr-147-fabric8-online-docs.badger.fabric8.io

File: docs/topics/modules/stopping_existing_workspaces.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (rkratky) [reply docs/topics/modules/stopping_existing_workspaces.adoc]: line break doesn't create a new paragraph, so it's useless.

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-147-6 is available for testing at http://fabric8-online-docs-app-pr-147-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-147-7 is available for testing at http://fabric8-online-docs-app-pr-147-fabric8-online-docs.badger.fabric8.io

**Pull request merged**